bin/workflows/pycbc_make_offline_search_workflow

```diff
@@ -523,9 +533,10 @@ for key in final_bg_files:
         closed_page = [(snrifar_cb, ifar_cb)]
     else:
         closed_page = [(snrifar_cb,)]
-    table = wf.make_foreground_table(workflow, bg_file, hdfbank, open_dir,
-                                     singles=insps, extension='.html',
-                                     tags=bg_file.tags)
+    if len(ifo_sets) > 1:
+        table = wf.make_foreground_table(workflow, bg_file, hdfbank, open_dir,
+                                         singles=insps, extension='.html',
+                                         tags=bg_file.tags)
```

Comment:
what exactly is the foreground table? something in the report? are we sure its something we only want if we have 3 or more ifos?

Reply:
I've added a comment to explain this - If there is only one ifo_set, then this table has already been made, so the workflow crashes given that two jobs are doing the same thing

Reply:
ok that makes sense